packages/core/handle/handle.js

```diff
@@ -695,6 +695,58 @@ const adaptEvent = handle.adaptEvent = curry(function (adapter, handler) {
 	}, 'adaptEvent');
 });
 
+/**
+ * Forwards the event to a function at `name` on `props`.
```

Comment:
How do you feel about changing this to something like:  Creates a handler that will forward ...